src/bin/observable.ts

```diff
@@ -123,14 +123,17 @@ try {
           "no-build": {
             type: "boolean",
             description: "Don’t build before deploying; deploy as is"
-          }
+          },
+          manifest: {type: "boolean", description: "Use the new upload strategy"},
+          "no-manifest": {type: "boolean", description: "Use the old upload strategy"}
         }
       });
       await import("../deploy.js").then(async (deploy) =>
         deploy.deploy({
           config: await readConfig(config, root),
           message,
-          force: build ? "build" : noBuild ? "deploy" : null
+          force: build ? "build" : noBuild ? "deploy" : null,
```

Comment:
The `--build` and `--no-build` arguments are already consolidated by the `helpArgs` helper. (Apologies for not noticing and fixing this earlier when I implemented this feature.) So, instead of pulling out the `noBuild` value, you can say:

```suggestion
          force: build === true ? "build" : build === false ? "deploy" : null,
```